tests/test.py

```diff
@@ -6243,59 +5664,6 @@ class TestTLSAuthenticationWithConnectorByDistinguishedName(TLSAuthenticationByD
     def setUp(self):
         if not TEST_WITH_TLS:
             self.skipTest("Test works only with TLS support on db side")
-        self.acra_writer_id = self.get_valid_certificate_identifier()
```

Comment:
why these tests remove and it named `TestTLSAuthenticationWithConnectorByDistinguishedName` (with connector:)